src/field_theory/is_alg_closed/basic.lean

```diff
@@ -518,7 +518,8 @@ begin
   haveI : is_scalar_tower F Fx A := is_scalar_tower.of_ring_hom (adjoin_root.lift_hom _ a ha),
   haveI : is_scalar_tower F Fx K := is_scalar_tower.of_ring_hom (adjoin_root.lift_hom _ x hx),
   haveI : fact (irreducible $ minpoly F x) := ⟨minpoly.irreducible $ this x⟩,
-  let ψ₀ : K →ₐ[Fx] A := is_alg_closed.lift (algebra.is_algebraic_of_larger_base F Fx hK),
+  let ψ₀ : K →ₐ[Fx] A,
```

Comment:
@eric-wieser there was also this one, that I don't understand completely. Note that the actual definition of `ψ₀` doesn't matter here, any morphism would do the job.